tests/dogfood.rs

```diff
@@ -49,17 +49,6 @@ fn dogfood_clippy() {
 #[test]
 fn dogfood_subprojects() {
     fn test_no_deps_ignores_path_deps_in_workspaces() {
```

Comment:
I think this function is ready to be on its own, instead of being an inner function.